src/libraries/System.Drawing.Common/tests/GraphicsTests.cs

```diff
@@ -102,6 +103,12 @@ public static IEnumerable<object[]> FromHdc_TestData()
         [MemberData(nameof(FromHdc_TestData))]
         public void FromHdc_ValidHdc_ReturnsExpected(IntPtr hdc)
         {
+            if (PlatformDetection.IsArm64Process && OperatingSystem.IsWindows())
+            {
+                // ActiveIssue: 51097
```

Comment:
Can we instead use the `ActiveIssue` that uses a conditional member?

https://github.com/dotnet/arcade/blob/a50c27610c6a060c677090b801636cd7149cd591/src/Microsoft.DotNet.XUnitExtensions/src/Attributes/ActiveIssueAttribute.cs#L23

If the condition is true, then it'll skip it. Here is an example: https://github.com/dotnet/runtime/blob/3319fa811bae3dedfbc952a2d7ed258886df7504/src/libraries/System.Dynamic.Runtime/tests/Dynamic.Declarations/Conformance.dynamic.declarations.formalParameter.Methods.cs#L211